build/README.md

```diff
@@ -180,33 +173,26 @@ following differences:
     original clusters public certs, to enable secure bi-directional
     communication.
 
-1.  All of the original clusters must then perform a rolling restart of their 
+1.  All of the original clusters must then perform a rolling restart of their
     cockroachdb pods to pick up the new certificates.
-    `kubectl rollout restart statefulset/cockroachdb --namespace dss-main`
-
+    `kubectl rollout restart statefulset/cockroachdb --namespace <NAMESPACE>`
 
 ## Using the CockroachDB web UI
 
-
 The CockroachDB web UI is not exposed publicly, but you can forward a port to
 your local machine using the kubectl command:
 
-
 ### Create a user account
 
-
 Pick a username and create an account:
 
-    kubectl -n dss-main exec cockroachdb-0 -ti -- \
+    kubectl -n $NAMESPACE exec cockroachdb-0 -ti -- \
```

Comment:
Let's maintain consistency and either change this to `<NAMESPACE>` to match above, or have the user set a $NAMESPACE variable above and carry $NAMESPACE all the way through (I'm indifferent to which approach is best, but we should stick to just one).